### ISSUE TITLE
[wish] Global shortcut

### ISSUE THREAD

**Issue by PAK90** [zadam/trilium#1046]
With the desktop (windows) app, it'd be wonderful to have a global shortcut that I can hit from anywhere, e.g. <kbd>shift</kbd> + <kbd>ctrl</kbd> + <kbd>z</kbd>. Then it'd bring up either the app, or a search/create UI of some sort. Basically it'd be nice to be able to create/edit/find notes at the drop of a hat,  with that global shortcut.

**Comment by zadam** [comment]
Actually there is a global default shortcut `CTRL-ALT-P` which will bring up the window and create a note under a daily note.

You can make other shortcuts global as well by prefixing them with `global:` in the shortcut options.

**Comment by PAK90** [comment]
That's rather awesome, thank you! This leads to another question though... for example, the Today button's shortcut is defined as `alt+t`, and that works fine. However, say I wanted to trigger a daily note from outside; I can't seem to do `global:alt+t`. Is there a way around that? Basically, is there a way of defining new global shortcut actions (i.e. actions, like the Today button, that aren't part of the Options menu)?

**Comment by zadam** [comment]
No, unfortunately that doesn't work ATM, but could be fixed. I'll put this into backlog.
